Makefile

```diff
@@ -91,8 +91,17 @@ deploy:
 	OUTPUT_CHARTS_DIR="$(OUTPUT_CHARTS_DIR)" \
 	./build/deploy.sh
 
+deploy-cf:
+	IMAGE="$(IMAGE)" \
+	TAG="$(TAG)" \
+	IMAGE_PULL_POLICY="$(IMAGE_PULL_POLICY)" \
+	OUTPUT_CHARTS_DIR="$(OUTPUT_CHARTS_DIR)" \
+	./build/deploy-cf.sh
```

Comment:
`deploy-cf.sh` appears to be nearly identical to `deploy.sh`; can we add options to `deploy.sh` to toggle between the two instead (to ensure that any fixes to one will be applied to the other)?

Not sure why, for example, `deploy-cf.sh` dropped `--recreate-pods`…

Reply:
`--recreate-pods` is deprecated, but I agree with you.

Reply:
Done with https://github.com/kubernetes-sigs/minibroker/pull/95/commits/e969dd074a5b5064e98d185ae1fb417f87116f96.